src/TheAlgorithms.jl

```diff
@@ -141,7 +141,8 @@ include("math/perfect_square.jl")
 include("math/sir_model.jl")
 include("math/sum_of_arithmetic_series.jl")
 include("math/sum_of_geometric_progression.jl")
-include("math/sieve_of_eratosthenes.jl")
+include("math/verlet.jl")
```

Comment:
it seems deleting this `include` has caused the codecov issues and failed tests. pls readd them